src/app/modules/group/components/group-join-by-code/group-join-by-code.component.html

```diff
@@ -23,14 +23,14 @@
       <span class="label red" *ngIf="!codeInfo.hasCodeNotSet && codeInfo.hasCodeExpired" i18n>
         This code has expired on {{ codeInfo.codeExpiration | date:'medium' }}
       </span>
-      <alg-button
-        label="Generate a code"
+      <p-button
```

Comment:
I didn't see a test case for this file :confused:

Reply:
I've add one more test case